components/contracts/dataGen.ts

```diff
@@ -216,12 +231,14 @@ export function generateUserCentric(
  * @param objectives The objectives.
  * @param gameChangers The game changers.
  * @param displayOrder The order in which to display the objectives.
+ * @param IsEvergreenSafehouse Is the contract the safehouse?
  * @returns The converted objectives.
  */
 export function mapObjectives(
     objectives: MissionManifestObjective[],
     gameChangers: string[],
     displayOrder: GroupObjectiveDisplayOrderItem[],
+    IsEvergreenSafehouse = false,
```

Comment:
How did this end up here?

Reply:
Oops, due to the switch-a-roo after moving it away from the evergreen branch :)